src/status_im/transport/handlers.cljs

```diff
@@ -52,6 +52,7 @@
                               {:db (assoc-in db [:transport/chats chat-id :sym-key-id] sym-key-id)
                                :shh/add-filter {:web3       web3
                                                 :sym-key-id sym-key-id
+                                                :one-to-one true
                                                 :topic      topic
                                                 :chat-id    chat-id}
                                :data-store.transport/save {:chat-id chat-id
```

Comment:
@cammellos Are you sure we don't want this information (`:one-to-one` flag set to true) persisted as well ?
You establish the filter with it, but don't include it in the `chat-transport-info` map which is persisted, so the next time app starts, I'm afraid the filter will be established without that flag.

Reply:
@janherich I will double check that to make sure, but I think it is persisted, I have added it in `create-chat` above, defaults to false but allow overriding (we set it to `true` for public chats)

Reply:
Here, https://github.com/status-im/status-react/blob/ac48f9edac136cca91b489783c8c691b382b34d5/src/status_im/transport/db.cljs#L32

Reply:
Ok, I see, it was already set correctly by previously and this just adds the filter.

Reply:
@janherich correct, this applies to the current develop, if we merge the changes to publish to pk this will have to be revisited.